### PR TITLE
BUG: longitude directory for non-integer longitudes

### PR DESCRIPTION
**Pull request by jklenzing**
# Description

The longitude directory name requires an integer when naming, but this is not required on the user.  This pull rounds off the chosen longitude to an integer, which should be sufficient for storing data and separating save sets.  A float value is still allowed for the actual run so that runs intersecting specific ground stations can still be generated.

## Type of change

- Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Running for an intersection with Jicamarca Radio Observatory:

```
import sami2py
lat = -11.9523
lon = -76.8758
sami2py.run_model(tag='jro', lat=lat, lon=lon, year=2012, day=1)
```

The develop branch will fail because it cannot create a directory from a float value.

**Test Configuration**:
* Mac OS X 10.15.7
* Python 3.8.2

# Checklist:

- [x] Make sure you are merging into the ``develop`` (not ``main``) branch
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] Add a note to ``CHANGELOG.md``, summarizing the changes

### CODE REVIEW

File: sami2py/utils.py [hunk 1]
```diff
@@ -79,7 +79,7 @@ def generate_path(tag, lon, year, day, test=False):
         str_fmt1 = 'lon{lon:03d}'
         str_fmt2 = '{year:4d}_{day:03d}'
         archive_path = os.path.join(top_directory, tag,
-                                    str_fmt1.format(lon=lon),
+                                    str_fmt1.format(lon=int(lon)),
```
Comment (JonathonMSmith) [inline sami2py/utils.py:82]: Only the number used to generate the archive directory is an int. So SAMI2 will run at the float longitude right?

**Review by JonathonMSmith — APPROVED** [review]

File: sami2py/utils.py (hunk 1 above)
Comment (jklenzing) [reply sami2py/utils.py:82]: that's right.  I should probably document that somewhere

**Comment by jklenzing** [comment]
Thanks @JonathonMSmith.  I've updated the docstring for `sami2py.utils.generate_path` to state the new behavior.  Does that look ok?

File: sami2py/utils.py (hunk 1 above)
Comment (JonathonMSmith) [reply sami2py/utils.py:89]: yup! I think that makes it clear

**Pull request merged**